Before: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
After: 6d29e0da667d482673380def736af0984c687568e56751ed43d61c2191204329
Patch: 0b5eb5238f1b1b17f4b4c45a34a0d78acbfa8a2e0e2c9dfd38a0daec16aee1b9
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ corvid-features/cold.rb	2012-09-19 17:43:56.000000000 +1000
@@ -0,0 +1,5 @@
+requirements 'fake:hot'
+install {
+  regenerate_template_with_feature 'example.txt.tt'
+}
+
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ corvid-features/hot.rb	2012-09-19 17:43:56.000000000 +1000
@@ -0,0 +1,3 @@
+install {
+  template2 'example.txt.tt'
+}
--- /dev/null	1980-01-01 00:00:00.000000000 +1000
+++ example.txt.tt	2012-09-19 17:43:56.000000000 +1000
@@ -0,0 +1,11 @@
+This is an example of an update.
+
+<% if feature_installed? :hot -%>
+  Hot day.
+<% end -%>
+
+<% if feature_installed? :cold -%>
+  Getting cold.
+<% end -%>
+
+Done! Updating works!
